augur/data/schema-export-v2.json

```diff
@@ -521,7 +521,7 @@
                                 }
                             },
                             "patternProperties": {
-                                "^[a-zA-Z0-9_-]+$": {
+                                "^[a-zA-Z0-9_.-]+$": {
```

Comment:
Would be good to add `*` here as well since it's present in the pattern for `genome_annotations` keys.